### DISCUSSION TITLE
ZIGBEE2MQTT Silicon Labs Multiprotocol Yellow assistant

### DISCUSSION THREAD

**Discussion by Benoit6303** [Koenkk/zigbee2mqtt#17454] [Devices]
Hello,

I have a yellow assistant with zibgee2mqtt and as soon as I flash to have the multiprotocol, it no longer works

I have to know what for port?
example :
tcp:192.168.1.100:9999
adapter: ezsp

I tried that to no avail, it won't start.

I had to reflash to find the use of the zigbee

THANKS

**Comment by Smart-Techno1ogy** [comment]
Hi, good morning. Did you ever get this to work? I am having the same "Failed to start" issue and have my config set as below. Are you able to assist in getting me up and running? 

Thanks 

![image](https://github.com/Koenkk/zigbee2mqtt/assets/16518539/31a81523-916d-494b-b628-d1b64756b93d)
